### PR TITLE
Amp Ads page

### PR DESCRIPTION
**Pull request by leeee**

### CODE REVIEW

**Comment by bpaduch** [comment]
I'm getting 404's for:
- /static/img/about/who-use-amp/amp-ads/lines_bottom2.svg
- /static/img/about/who-use-amp/amp-ads/lines_bottom1.svg

I think you need to update the following with your new images: https://github.com/ampproject/docs/blob/master/content/learn/who/advertisers.yaml

**Comment by leeee** [comment]
Good catch, fixed!

**Comment by bpaduch** [comment]
Staged here: https://ampproject-custom.firebaseapp.com/learn/who-uses-amp/amp-ads/

**Comment by bpaduch** [comment]
cc: @amedina

File: assets/img/about/who-use-amp/amp-ads/amp_logo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0,0,1,.28,0h2.86V24.12H76.34v-8q0-.58.06-1.26L72.68,22a1.22,1.22,0,0,1-.47.51,1.3,1.3,0,0,1-.67.17H71.1a1.3,1.3,0,0,1-.67-.17A1.22,1.22,0,0,1,70,22l-3.74-7.15q0,.33.06.66t0,.61v8H63.43V10.24h2.86a.79.79,0,0,1,.28,0,.65.65,0,0,1,.23.15,1.17,1.17,0,0,1,.2.28l3.61,7.05q.19.36.37.75"/><path class="cls-1" d="M87.48,17h1.7a2.4,2.4,0,0,0,1.83-.6,2.37,2.37,0,0,0,.56-1.68,2.49,2.49,0,0,0-.14-.87,1.78,1.78,0,0,0-.44-.67,2,2,0,0,0-.74-.43,3.31,3.31,0,0,0-1.06-.15h-1.7V17Zm0,2.43v4.64H84.22V10.23h5a8.25,8.25,0,0,1,2.55.35,4.9,4.9,0,0,1,1.75,1,3.74,3.74,0,0,1,1,1.44,4.83,4.83,0,0,1,.33,1.79,5.23,5.23,0,0,1-.34,1.91,3.84,3.84,0,0,1-1,1.49,4.87,4.87,0,0,1-1.76,1,8.13,8.13,0,0,1-2.52.35Z"/><path class="cls-1" d="M22.24,15,15.35,26.37H14.1L15.33,19H11.46a.62.62,0,0,1-.62-.62A1.14,1.14,0,0,1,11,18L17.84,6.62h1.27L17.84,14h3.9a.62.62,0,0,1,.62.62.6.6,0,0,1-.13.36ZM16.61.23A16.27,16.27,0,1,0,33,16.5,16.35,16.35,0,0,0,16.61.23Z"/><path class="cls-1" d="M50.58,18.81h3.83L53.12,15l-.29-.85q-.17-.49-.33-1.06-.15.58-.32,1.08t-.31.86l-1.29,3.78Zm9,5.31H57.07a1.18,1.18,0,0,1-.7-.2,1,1,0,0,1-.38-.5L55.16,21H49.83L49,23.43a1.07,1.07,0,0,1-.38.48,1.09,1.09,0,0,1-.68.21H45.39l5.43-13.89h3.33l5.43,13.89Z"/></svg>
```
Comment (pbakaus) [inline assets/img/about/who-use-amp/amp-ads/amp_logo.svg]: Why can't we reuse the other AMP logo?

File: scripts/update_blog_ads_links.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env node
```
Comment (pbakaus) [inline scripts/update_blog_ads_links.js]: This whole file looks very much like the other script. What are the differences? Can we not organize it into one script?

**Comment by bengotow** [comment]
Glad to see @googlebot's got my back — definitely cool with my commits being included. 

I'll merge the `update_blog_ads_links.js` script into the other one this evening. Was torn between keeping this new ads site isolated and refactoring it a bit, but I definitely think it'd be cleaner that way. Stay tuned!

**Comment by bengotow** [comment]
Hey @pbakaus — I've pushed a commit that addresses the build script duplication, let me know what you think. There was some duplicate code in there already for handling the blog and video streams and I needed to introduce slight changes in a few places to handle the ads feed,  so I refactored it a good bit and added comments.

To make sure I didn't break anything, I ran the old scripts and committed the generated files to a test branch, patched in the new script and ran it, and confirmed that the generated files are identical.

File: assets/img/about/who-use-amp/amp-ads/amp_logo.svg (hunk 1 above)
Comment (leeee) [inline assets/img/about/who-use-amp/amp-ads/amp_logo.svg]: We actually aren't even using this, removed

**Comment by leeee** [comment]
Hey @pbakaus, I added the redirect and made sure all the images are under 1MB. Let me know if you want any other changes!

**Comment by pbakaus** [comment]
Looks good, thanks all!

**Pull request merged**